### ISSUE TITLE
Not able to compile

### ISSUE THREAD

**Issue by rolfrm** [rabbibotton/clog#24]
Could not get this to work.
commit: b79a3431630efc285fe0293bdf7a3e196159801b
OS: Ubuntu 20.04.1
SBCL: 2.0.1.debian

[package clog-user]
; 
; caught ERROR:
;   READ error during COMPILE-FILE:
;   
;     Symbol "UPDATE-ASDF-SYSTEM-HTML-DOCS" not found in the MGL-PAX package.
;   
;       Line: 130, Column: 39, File-Position: 3658
;   
;       Stream: #<SB-INT:FORM-TRACKING-STREAM for "file ...../clog/clog-helpers.lisp" {1005EC2CB3}>

**Comment by rabbibotton** [comment]
It looks like an issue with dependencies. Try this first to update your quicklisp - (ql:update-dist "quicklisp").

**Comment by aykaramba** [comment]
Confirmed as well, exact same error.  Updated the dist just now.  Not sure how to go about addressing the issue.

**Comment by rolfrm** [comment]
I got it fixed by doing quicklisp upgrade as @rabbibotton suggested.

@aykaramba, did you try this?

**Comment by rabbibotton** [comment]
I will have to see if can find a debian machine, but for now can just comment out the line as you don't need to generate as the manual is checked in under doc/

**Comment by rabbibotton** [comment]
It seems to be a generic configuration issue, so closing ticket.

**Comment by aykaramba** [comment]
Well, if anyone else gets here with the same error, additional troubleshooting info:

1) ORIGINAL: As per my original post, yes I update the dist and the error still happened.

2) WORKS: I tried it on a new debian vm, clean sbcl install and clog installs just fine and seems to work.

3) DOESN'T WORK: Updating my local dist, uninstalling / reinstalling packages does absolutely nothing.  My quicklisp is NOT customized in any way, it has only ever had packages installed and removed and upgrades applied.  I did install borodusts bodge and ultralisp dists to get bodge and weblocks working, but that was only using their installation instructions, again no customization of any sort.

4) DOESN'T WORK: I made sure that I don't have Clog in my local-projects folder.  It is only installed in the root quicklisp directories.

5) WORKS:  As the author suggested, i commented out like 562 in this file: quicklisp/dists/quicklisp/software/clog-20210124-git/clog.lisp

And I was able to get clog to work on my main working quicklisp install.

6) CONCLUSIONS:

a)  Commenting out line 562 resolves the problem.

b) Given the above, I suspect that if I nuked my local quicklisp directory and installed from scratch that the issue would be resolved.  I am not about to do that, obviously, because I don't want to be reinstalling anything.

The author is correct to close the ticket as this is a config issue.  The solution lies somewhere between me understanding more about how quicklisp works and perhaps some quicklisp config utilities that would help provide more insight into these kinds of issues.  This isn't the first time that I have run into a package that refuses to install and we get dropped to a debugger ... which is okay but we can do better.

Thanks for the feedback everyone.
